src/localization/Code Strings.json

```diff
@@ -262,6 +262,14 @@
         "description": "",
         "message": "Keywords:"
     },
+    "book.metadata.rebrand": {
+        "description": "",
```

Comment:
I have the feeling a translator is going to ask for context on this one. We probably need to help them understand what a "rebrand" is. It might be sufficient to point them at book.metadata.rebrandMessage.